kasket.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.3.0'
 
-  s.add_runtime_dependency("activerecord", ">= 4.2", "< 5.2")
```

Comment:
is `< 6.0` a hard limit? Have you tried testing with 6?
Just hoping we could get that for free here